src/sentences/bwc.rs

```diff
@@ -149,7 +149,7 @@ mod tests {
 
         assert_eq!(
             BwcData {
-                fix_time: Some(NaiveTime::from_hms(8, 18, 37)),
+                fix_time: NaiveTime::from_hms_opt(8, 18, 37),
```

Comment:
I would suggest to do an `.expect()` on the returned `Option` and wrap it in `Some`. This way if the expected time is wrong in the test itself we will now.

Same goes for all other instances.

Reply:
Something like this ?
`fix_time: Some(NaiveTime::from_hms_opt(8, 18, 37).expect("invalid time"))`

Reply:
Yes, exactly!